### ISSUE TITLE
Flake8 rule 'E999' reported an invalid point

### ISSUE THREAD

**Issue by GrierPhillips** [AtomLinter/linter-flake8#549]
Flake8 reported an invalid point for the rule `E999`, with the messge `SyntaxError: invalid syntax`.  It claims the error is on the first line `'""Model Runner.`
```python
"""Model Runner.

This script consumes a run.json file and executes the contained model scenario.
"""

import json
import logging
import subprocess
from time import time
from typing import Dict

from lxml import etree
from pymongo import MongoClient

from errors import LaunchError
...
```

Debug information:
Atom version: 1.22.1
Flake8 version: `3.5.0 (flake8-bugbear: 17.4.0, flake8-pyi: 17.3.0, mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.6.0) CPython 3.6.3 on Darwin`

**Comment by Arcanemagus** [comment]
Unfortunately it is only reporting on the first line since, by definition, the point that `flake8` reported doesn't exist so it can't use that point.

Are you able to consistently reproduce this? If so, are you able to share the file (if not, can you create a minimal case)?

There was a previous bug in `flake8` that was fixed in v3.2.1, but there seems to be at least one more. Please follow the linked issue for any updates (and post a file to reproduce with there please, thanks!).

**Comment by Arcanemagus** [comment]
Duplicate of #505
